README.md

```diff
@@ -1,41 +1,12 @@
-# TIS Microservice Template
+# TIS Common Upload
 
 ## About
-This is a template to be used for TIS microservices with the following
-technology:
-
- - Java 11
- - Spring Boot
- - Gradle
- - JUnit 5
-
-Boilerplate code is to be generated with:
- - Lombok
- - MapStruct
-
-Code quality checking and enforcement is done with the following tools:
- - EditorConfig
- - Checkstyle
- - JaCoCo
- - SonarQube
-
-Error and exception logging is done using Sentry.
+TIS common upload to be used for upload files in S3.
 
 ## TODO
-To use this template, create a new repository from it and follow the TODOs in
-the code, with the following additional changes.
- - Update copyright year in [LICENSE](LICENSE).
- - Update copyright year in [TemplateApplication].
- - Update copyright year in [TemplateApplicationTest].
- - Update this README.
- - Set up Sentry project.
  - Provide `SENTRY_DSN` and `SENTRY_ENVIRONMENT` as environmental variables
    during deployment.
- - Add repository to SonarCloud.
- - Add SonarCloud API key to repository secrets.
- - Add repository to Dependabot.
```

Comment:
Leave the TODOs for Sentry/Sonar/Dependabot in the README until they are done, same with any others not fully covered in this PR.

Reply:
addressed